loom/engine/bindings/loom/lmGameController.h

```diff
@@ -0,0 +1,35 @@
+#include <SDL.h>
+
+#include "loom/script/loomscript.h"
+
+#define MAX_CONTROLLERS 4
+
+class LoomGameController
```

Comment:
Diff does appear to be out of date, yes. https://github.com/LoomSDK/LoomSDK/blob/TO-JoystickIntegration/loom/engine/bindings/loom/lmGameController.cpp